lib/client/api.go

```diff
@@ -1192,7 +1408,36 @@ func NewClient(c *Config) (tc *TeleportClient, err error) {
 		// if the client was passed an agent in the configuration and skip local auth, use
 		// the passed in agent.
 		if c.Agent != nil {
-			tc.localAgent = &LocalKeyAgent{Agent: c.Agent, keyStore: noLocalKeyStore{}, siteName: tc.SiteName}
+			webProxyHost, _ := tc.WebProxyHostPort()
```

Comment:
What's the _ that you're ignoring here?

Reply:
That's the port (int). That said we also have a `WebProxyHost()` helper that does exactly this, so I'll switch to it.